mne/decoding/transformer.py

```diff
@@ -497,6 +497,10 @@ class FilterEstimator(TransformerMixin):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
         Defaults to self.verbose.
+
```

Comment:
I'm actually for deprecating this class, it's completely redundant. @agramfort @jasmainak WDYT?